test/appium/tests/medium/test_chats_m.py

```diff
@@ -739,6 +739,53 @@ def prepare_devices(self):
         for i in range(1, 3):
             self.chats[i] = self.homes[i].get_chat(self.chat_name).click()
 
+    @marks.testrail_id(702343)
+    def test_group_chat_send_delete_image(self):
+
+        self.chats[0].just_fyi("Sending image to group chat")
+        self.chats[0].show_images_button.click()
+        self.chats[0].allow_button.click()
+        self.chats[0].first_image_from_gallery.click()
+        self.chats[0].send_message_button.click()
+
+        self.chats[0].just_fyi("Verify sent image is displayed for every group member")
+        for i in range(1, 3):
+            if not self.chats[i].image_message_in_chat.is_element_displayed(20):
+                self.errors.append("Sent image is not displayed in chat for user driver '%s'" % i)
+
+        self.chats[0].just_fyi("Deleting image message from group chat")
+        self.chats[0].image_message_in_chat.long_press_element()
+        self.chats[0].element_by_translation_id("delete").click()
+
+        self.chats[0].just_fyi("Verify deleted image is NOT displayed for every group member")
+        for i in range(1, 3):
+            if self.chats[i].image_message_in_chat.is_element_displayed(20):
```

Comment:
There is special method for such cases is_element_disappeared() which is waiting until element will disapper; otherwise it may fail quite often, as messages won't disappear immidiately

Reply:
done